test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,24 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertNoNull();
+        // no null
+        assertNoNull("a", "b", Integer.valueOf(1));
+        assertNoNull(1, 2, 3, Integer.valueOf(4), 5);
+        // some are null
+        assertSomeAreNull((Object) null);
+        assertSomeAreNull(null, null);
+        assertSomeAreNull(1, 2, null, 4);
+        assertSomeAreNull("a", 2, 3, null, "e");
+    }
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+    private void assertSomeAreNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
```

Comment:
In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch